lib/explorer/data_frame.ex

```diff
@@ -1642,6 +1642,44 @@ defmodule Explorer.DataFrame do
        when is_atom(colname) and is_map(acc) and is_list(value),
        do: mutate_reducer({Atom.to_string(colname), value}, acc, df)
 
+  @doc """
+   Display the DataFrame in a tabular fashion.
+
+   ## Examples
+
+       df = Explorer.Datasets.fossil_fuels()
+       Explorer.DataFrame.table(df)
```

Comment:
I think `Explorer.Datasets.iris/0` is sufficiently small that it would make sense to put it here and make the doctest functional.

Reply:
@cigrainger I'm having problems at making the doctest work.
Can you do this part?

Reply:
Sure!

Reply:
Oh... I'm sorry I was being silly. Doctests only accept valid Elixir syntax or opaque data starting with `#`. Perfectly fine to leave it without the functional doctest. I'll add a normal test.